src/store/actions.js

```diff
@@ -65,7 +68,13 @@ export default {
    * }
    */
   executeTurn(context, payload) {
+   
     bus.$emit('card-played')
+    log.warn({name:payload.player.name, card_played:payload.card.type, value: payload.card.value})
```

Comment:
I think now that you have added the log calls to the individual turn actions  this could be removed along with the extra blank lines below it.

Reply:
Oh yes, I suppose not all turns are covered by the other functions you are logging.

Reply:
Then just removing the extra blank lines is all to do here